AuxTel/AuxTel-Troubleshooting/LATISS-Troubleshooting/ATspectrograph-failed.rst

```diff
@@ -0,0 +1,150 @@
+    Traceback (most recent call last): File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-7.0.1/lib/python3.11/site-packages/lsst/ts/salobj/base_script.py",
+    line 603, in do_run await self._run_task File "/net/obs-env/auto_base_packages/ts_standardscripts/python/lsst/ts/standardscripts/auxtel/daytime_checkout/latiss_checkout.py", line 110, in run await self.latiss.setup_instrument( File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/latiss.py", line 176, in setup_instrument await self.setup_atspec( File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/latiss.py", line 242, in setup_atspec await asyncio.gather(*setup_coroutines) File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-7.0.1/lib/python3.11/site-packages/lsst/ts/salobj/topics/remote_command.py", line 416, in set_start return await self.start(timeout=timeout, wait_done=wait_done) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-7.0.1/lib/python3.11/site-packages/lsst/ts/salobj/topics/remote_command.py", line 487, in start return await cmd_info.next_ackcmd(timeout=timeout)
+ 
+ 
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-7.0.1/lib/python3.11/site-packages/lsst/ts/salobj/topics/remote_command.py",
+    line 205,
+    in next_ackcmd raise base.AckTimeoutError( lsst.ts.salobj.base.AckTimeoutError: msg='Timed out waiting for command acknowledgement',
+    ackcmd=(ackcmd private_seqNum=1137560160, ack=<SalRetCode.CMD_NOACK: -301>, error=0, result='No command acknowledgement seen')
+
+.. _ATspectrograph-failed/Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+
+.. the link below gets an "This site can’t be reached"
+
+#. Power cycle the ATSpectrograph
+    #. Check ATSpectrograph CSC is in Standby Status.
+    #. Connect to ``http://aux-pdu-spectrograph.cp.lsst.org/``
+    #. Log in with Username/PW (it is in the AuxTel 1Password vault).
+    #. Click On/Off button for **Outlet 2** only. 
+
+   .. image:: ./_static/1_power_cycle_ATSpec.png
+      :width: 700px
+
+#. Connect to AuxTel EUI desktop
+    #. **Install Microsoft Remote Desktop**
```

Comment:
This part is now available in the AuxTel/AuxTel-Non-standard-Operations/AuxTel-EUI-Access/AuxTel-EUI-Access.rst, 
so it would be better to include an internal reference to that page. Thanks!

Reply:
@isotuela, does it mean that the entire step 2 will be in another page, correct?